src/virtualize.js

```diff
@@ -132,7 +132,16 @@ export default function virtualize(MyComponent) {
       }
 
       this.setState(nextState, () => {
-        this.setWindowTimer();
+        // We are going backward, but reached first rendered item, let's render previous right away, with no timer
+        if (indexDiff < 0 && index > 0 && indexContainer === 0) {
```

Comment:
This issue can happen when the index is below zero. It would be better to remove it.